plugins/jenkins_status_checker/app/views/samson_jenkins_status_checker/_deploys_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if ENV['JENKINS_STATUS_CHECKER'] && stage.production? && project.show_jenkins_status_checker %>
```

Comment:
can it use `@stage` here and then remove the `stage: @stage` passing logic in app/views/deploys/new.html.erb ?